modules/ducktests/tests/ignitetest/tests/thin_client_query_test.py

```diff
@@ -70,26 +70,33 @@ def test_thin_client_index_query(self, server_version, filter):
         ignite.stop()
 
 
-class IgniteNodeSpecExcludeDucktests(IgniteNodeSpec):
-    """
-    Ignite node specification that excludes module 'ducktests' from classpath.
-    """
-    def modules(self):
+def return_spec_without_ducktests(service, base_spec):
```

Comment:
Please add description of the function and parameters

Reply:
done